.github/workflows/main.yaml

```diff
@@ -123,19 +123,17 @@ jobs:
         if: matrix.os == 'windows-latest'
         shell: cmd
         run: |
-          choco install llvm
-        # run: |
-        #   curl --proto '=https' --tlsv1.2 -sSf https://github.com/llvm/llvm-project/releases/download/llvmorg-10.0.0/LLVM-10.0.0-win64.exe -L -o llvm-installer.exe
-        #   7z x llvm-installer.exe -oC:/llvm-10
-        #   echo C:/llvm-10/bin >> $GITHUB_PATH
-        #   echo "LLVM_SYS_100_PREFIX=C:/llvm-10" >> $GITHUB_ENV
-        #   echo "LIBCLANG_PATH=C:/llvm-10/bin/libclang.dll" >> $GITHUB_ENV
-      - name: Install LLVM (Unix)
+          curl --proto '=https' --tlsv1.2 -sSf ${{ matrix.llvm_url }} -L -o llvm.tar.gz
+          7z x llvm.tar.gz -oC:/llvm-10
+          echo C:/llvm-10/bin >> $GITHUB_PATH
+          echo "LLVM_SYS_100_PREFIX=C:/llvm-10" >> $GITHUB_ENV
+          echo "LIBCLANG_PATH=C:/llvm-10/bin/libclang.dll" >> $GITHUB_ENV
```

Comment:
The only difference between Windows and Linux+macOS here is the location where LLVM is uncompressed. Note that `7z` is available on all platforms, and `tar xzvf … -C …` also works on all platforms.

Consequently, we may want to remove the `if:` step's condition, and merge the two steps into a single one.

Thoughts?

Reply:
Oh yes, if we can have the exact same step on every platform, it could be great, will have a look at that.

Thanks for the hint.